packages/volto/Makefile

```diff
@@ -31,243 +31,256 @@ YELLOW=`tput setaf 3`
 # Top-level targets
 
 .PHONY: all
-all: build
+all: help
```

Comment:
@sneridagh is this right? all used to build now it points to help. Why was all in the first place?

Reply:
@ichim-david you could look at the commit history to find when it was added, and there might be a useful commit message. Else it was probably a convention that was copy-pasted from who knows where.